### PR TITLE
fix: Remove column` requestJson` and add column `user_id` in stack_analyses_request table

### PR DESCRIPTION
**Pull request by rafiu007**
Add another column user_id to the stack_analyses_request table .This will be used to persist the uuid.
Drop column requestJson  as it seems to have the same data as dep_snapshot from stack_analyses_request table.

### CODE REVIEW

**Comment by rafiu007** [comment]
[test]

File: f8a_worker/models.py [hunk 1]
```diff
@@ -362,13 +362,13 @@ class StackAnalysisRequest(Base):
     __tablename__ = "stack_analyses_request"
     id = Column(String(64), primary_key=True)
     submitTime = Column(DateTime, nullable=False)
-    requestJson = Column(JSON, nullable=False)
     origin = Column(String(64), nullable=True)
     result = Column(JSON, nullable=True)
     team = Column(String(64), nullable=True)
+    user_id = Column(String(64),nullable=False)
+    dep_snapshot = Column(JSONB, nullable=True)
```
Comment (arajkumar) [inline f8a_worker/models.py:369]: Follow the existing coding pattern, It seems it uses camelCase instead of snake_case.

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: f8a_worker/models.py (hunk 1 above)
Comment (arajkumar) [reply f8a_worker/models.py:369]: It looks like both type of column names are used. Ignore my comment.

Comment (rafiu007) [reply f8a_worker/models.py:369]: @arajkumar  dep_snapshot was already in snake case and also used in other repos where this is imported.DO you want me to change them all?

File: f8a_worker/models.py [hunk 2]
```diff
@@ -362,13 +362,13 @@ class StackAnalysisRequest(Base):
     __tablename__ = "stack_analyses_request"
     id = Column(String(64), primary_key=True)
     submitTime = Column(DateTime, nullable=False)
-    requestJson = Column(JSON, nullable=False)
     origin = Column(String(64), nullable=True)
     result = Column(JSON, nullable=True)
     team = Column(String(64), nullable=True)
+    user_id = Column(String(64),nullable=False)
```
Comment (arajkumar) [inline f8a_worker/models.py:368]: user_id will be a type of uuid, is there a sql native type for uuid?

File: f8a_worker/models.py (hunk 1 above)
Comment (arajkumar) [reply f8a_worker/models.py:369]: Ignore this comment.

File: alembic/README.md [hunk 3]
```diff
@@ -11,5 +11,6 @@ You need to run the alembic command from the directory in which alembic.ini is l
 3. Enable port forwarding to the DB instance in your dev cluster. This is required because alembic needs to find the 
    difference between the schema in models.p and the database 
     while [ 1 ]; do oc port-forward `<your pgbouncer pod id`> 5432:5432; done
-4. Run alembic revision --autogenerate -m "`<your message`>"
+4. Add `export F8A_POSTGRES=postgresql://localhost:5432/coreapi?password=<yourpassword>&user=coreapi` to your .bashrc and source it.
```
Comment (arajkumar) [inline alembic/README.md:14]: No need to add this to bashrc., you can just simply export this in to the current shell.

File: alembic/env.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """Script to run the database migrations, online or offline."""
 from __future__ import with_statement
 
-import os
+import os
```
Comment (arajkumar) [inline alembic/env.py:4]: Remove this change

File: alembic/versions/0809bdb23579_ading_user_id.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
+"""ading user_id.
```
Comment (prashbnair) [inline alembic/versions/0809bdb23579_ading_user_id.py:1]: It will be good to add a proper comment when creating this file as it is useful.

**Review by prashbnair — APPROVED** [review]

**Comment by sawood14012** [comment]
[test]

**Comment by rafiu007** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Review by arajkumar — APPROVED** [review]
lgtm

**Comment by prashbnair** [comment]
[test]

**Pull request merged**